Meta/lint-keymaps.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import sys
+
+
+PERMITTED_MAPS = ['map', 'shift_map', 'alt_map', 'altgr_map', 'shift_altgr_map']
+REQUIRED_MAPS = ['map', 'shift_map', 'alt_map']
+# See Userland/Libraries/LibKeyboard/CharacterMapFile.cpp
+# and Userland/Libraries/LibKeyboard/CharacterMap.cpp.
+GOOD_MAP_LENGTHS = {90, 128}
+
+
+def report(filename, problem):
+    print('{}: {}'.format(filename, problem))
+
+
+def validate_single_map(filename, mapname, values):
+    all_good = True
+
+    if not isinstance(values, list):
+        report(filename, '"{}" is not a list'.format(mapname))
+        return False  # Cannot continue other checks
+
+    if not any(values):
+        report(filename, 'no values set in {}'.format(mapname))
+        all_good = False
+
+    for i, c in enumerate(values):
+        if len(c) > 1:
+            report(filename, 'more than one character (“{}”) for charmap index {} of {}'.format(c, i, mapname))
```

Comment:
Can we just use "normal" quotes here?

```suggestion
            report(filename, 'more than one character ("{}") for charmap index {} of {}'.format(c, i, mapname))
```

Reply:
But but but … *fancy*!

Yes, absolutely, I changed it :)